pkg/controller/pxc/service_test.go

```diff
@@ -0,0 +1,213 @@
+package pxc
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc"
+)
+
+var _ = Describe("Service Labels Annotations Test", Ordered, func() {
```

Comment:
You can name it only like `Service labels and annotations`.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1517/commits/64789d32c0277ee4d8717000206978deb28f3a71